components/styles.js

```diff
@@ -73,8 +73,8 @@ const styles = StyleSheet.create({
     backgroundColor: 'transparent',
     width: 28,
     height: 28,
-    left: 13,
-    top: 13,
+    left: (DEVICE_WIDTH / 14) - 14,
```

Comment:
oh okay I remember talking about this position: 'absolute' in the last PR now lol